src/controllers/controller.polarArea.js

```diff
@@ -120,13 +120,7 @@ export default class PolarAreaController extends DatasetController {
 		}
 
 		// Scriptable options
-		const context = {
-			chart: me.chart,
-			dataPoint: this.getParsed(index),
-			dataIndex: index,
-			dataset,
-			datasetIndex: me.index
-		};
+		const context = me.getContext(index, false);
```

Comment:
What's the reason that this uses `false` vs the bubble controller that uses `mode === 'active'` for the 2nd parameter?

Reply:
No reason really, updated.
The `mode === 'active'` is strictly not correct either, because the mode is kept active when going back from active to normal (so the `active` animation configuration is used).

I think we need to wait for more feedback to see how the animations work in different scenarios before going further though.